base_geoengine/security/data.xml

```diff
@@ -1,15 +1,14 @@
-<openerp>
-    <data>
-    
-        <record id="group_geoengine_user" model="res.groups">
-            <field name="name">Geoengine user</field>
-        </record>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
 
-        <record id="group_geoengine_admin" model="res.groups">
-            <field name="name">Geoengine admin</field>
-            <field name="implied_ids" eval="[(4, ref('group_geoengine_user')),]"/>
-            <field name="users" eval="[(4, ref('base.user_root'))]"/>
-        </record>
+    <record id="group_geoengine_user" model="res.groups">
+        <field name="name">Geoengine user</field>
+    </record>
 
-    </data>
-</openerp>
\ No newline at end of file
+    <record id="group_geoengine_admin" model="res.groups">
+        <field name="name">Geoengine admin</field>
```

Comment:
s/admin/Admin

Reply:
I don't understand what you mean.

Reply:
@benwillig  I think the idea is to capitalize de "a" of "admin" (s/old_pattern/new_pattern) :smile:

Reply:
Thanks!